library/src/main/java/com/sothree/slidinguppanel/SlidingUpPanelLayout.java

```diff
@@ -527,19 +527,16 @@ public void setDragView(View dragView) {
             mDragView.setClickable(true);
             mDragView.setFocusable(false);
             mDragView.setFocusableInTouchMode(false);
-            mDragView.setOnClickListener(new OnClickListener() {
```

Comment:
Please keep this way. The auto Kotlib converter comes with issues by using lambdas